pym/bob/cmds/build/builder.py

```diff
@@ -33,6 +33,10 @@
 import stat
 import sys
 import tempfile
+import threading
+
+if sys.platform is not "win32":
```

Comment:
While it will technically work should compare by value (`==`) and not by identity.